main/modes/system/intro/introMode.c

```diff
@@ -498,44 +521,57 @@ static void introExitMode(void)
  */
 static void introMainLoop(int64_t elapsedUs)
 {
-    // clearPxTft();
-
 #ifdef CUSTOM_INTRO_SOUND
     if (iv->playingSound || !iv->introComplete)
     {
-        int64_t songTime     = (iv->soundSize * 1000000 + DAC_SAMPLE_RATE_HZ - 1) / DAC_SAMPLE_RATE_HZ;
-        int64_t animTime     = (songTime * 2 / 3);
-        static int64_t timer = 0;
+        for (int i = 0; i < TFT_HEIGHT; i++)
+        {
+            drawLineFast(0, i, TFT_WIDTH, i, (i % 2) ? c555 : c444);
```

Comment:
Not that it matters, but an effect like this is faster to paint if you get the framebuffer with `getPxTftFramebuffer()` and using `memset()` to draw the bands. Lines are cool too